src/pages/my-cooperations/MyCooperations.styles.ts

```diff
@@ -14,5 +14,7 @@ export const styles = {
     borderBottom: '1px solid',
     borderColor: 'primary.100'
   },
+  contentContainer: { minHeight: '100vh' },
```

Comment:
it doesn't solve the problem, we will get a lot of empty space when there would not be enough content

Reply:
fixed, used redux to hide footer